packages/clay-date-picker/src/DateNavigation.js

```diff
@@ -0,0 +1,154 @@
+import * as Helpers from './Helpers';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React, {useMemo} from 'react';
+
+import Button from './Button';
+import Select from './Select';
+
+/**
+ * @param {Object} param
+ * @return {React.createElement}
+ */
+function DateNavigation({
+	currentMonth,
+	months,
+	onDotClicked,
+	onMonthChange,
+	spritemap,
+	years,
+}) {
+	const memoizedYears = useMemo(
+		() =>
+			Helpers.range(years).map(elem => {
+				return {
+					label: elem,
+					value: elem,
+				};
+			}),
+		[years.start, years.end]
+	);
+
+	const memoizedMonths = useMemo(
+		() =>
+			months.map((month, index) => {
+				return {
+					label: month,
+					value: index,
+				};
+			}),
+		months
```

Comment:
does this need to be in brackets?

Reply:
Too, but as they are array would have no problem. But I'll add in brackets to keep a pattern here.